package.json

```diff
@@ -42,23 +42,29 @@
     "react-router-dom": "^5.1.2",
     "react-static": "^7.2.2",
     "react-static-plugin-react-router": "^7.2.2",
+    "react-test-renderer": "^16.11.0",
```

Comment:
Probably this and storybook are dev dependencies

Reply:
right again. updating now